tests/TransferStatsTest.php

```diff
@@ -0,0 +1,29 @@
+<?php
+namespace GuzzleHttp\Tests;
+
+use GuzzleHttp\TransferStats;
+use GuzzleHttp\Psr7;
+
+class TransferStatsTest extends \PHPUnit_Framework_TestCase
+{
+    public function testHasData()
```

Comment:
In the other tests, classes are named SomethingTest, the functions are called testSomething, so I think this is OK. (Not really sure, but that is what the file tests/ClientTest.php has in its code).